wordpress-mobile/WordPress-iOS
# Safely unwrap searchController in Posts List

Fixes #13735

It seems like the `searchController` property can be `nil` in cases where the view controller is quickly dismissed during loading pre-iOS 13.

I decided to safely unwrap this and _not_ change the behavior where the Quick Actions buttons set a new detail view controller on each tap. Selecting a table view cell behaves differently by doing nothing if that screen is already shown in the detail pane (since the selected table view index does not change). I think this change should still be made but the button behavior could be changed separately.

To test:
- Tap the Posts Quick Action button (in the Blog Details view) several times rapidly on an iPad running iOS 12 (simulator works fine).
- Notice that the app does not crash in this PR, but did crash before

PR submission checklist:
